lib/plugins/aws/lib/wait-for-change-set-creation.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const wait = require('timers-ext/promise/sleep');
+const ServerlessError = require('../../../serverless-error');
+const isChangeSetWithoutChanges = require('../utils/is-change-set-without-changes');
+const { log } = require('@serverless/utils/log');
+
+module.exports = {
+  async waitForChangeSetCreation(changeSetName, stackName) {
+    const params = {
+      ChangeSetName: changeSetName,
+      StackName: stackName,
+    };
+
+    const callWithRetry = async () => {
+      const changeSetDescription = await this.provider.request(
+        'CloudFormation',
+        'describeChangeSet',
+        params
+      );
+      if (
+        changeSetDescription.Status === 'CREATE_COMPLETE' ||
+        isChangeSetWithoutChanges(changeSetDescription)
+      ) {
+        return changeSetDescription;
+      }
+
+      if (
+        changeSetDescription.Status === 'CREATE_PENDING' ||
+        changeSetDescription.Status === 'CREATE_IN_PROGRESS'
+      ) {
+        log.info('Change Set did not reach desired state, retrying');
+        await wait(5000);
```

Comment:
It'll be nice to resolve value same way as we do for stack monitoring (so with support for `process.env.SLS_AWS_MONITORING_FREQUENCY `) - we may exclude it to some util

Reply:
good idea 👍